### PR TITLE
Added link to ESDS governance document to About page.

### PR DESCRIPTION
**Pull request by clyne**
This PR adds a short paragraph to the About page, and includes a link to our governance document.

N.B. we should probably include a code of conduct at some point

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # About Us
 
+The Earth Systems Data Science (ESDS) initiative is a community
```
Comment (katiedagon) [inline about.md:3]: We usually write System as singular (not Systems) in ESDS.

File: about.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+with a common interest in advancing geoscientist’s ability to make
+effective use of modern, open tools and technologies to support
+their Data Science needs. ESDS welcomes participation from all
+NCAR/UCAR LCPOs. We are primarily a volunteer organization, and
```
Comment (katiedagon) [inline about.md:7]: Should define LCPO.

**Review by katiedagon — COMMENTED** [review]
Thanks @clyne for adding this. I agree about adding a code of conduct at some point.

**Comment by clyne** [comment]
Thanks, Katie. I added what might be the only definition of LCPO that exists :-)

File: about.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # About Us
 
+The Earth System Data Science (ESDS) initiative is a community
+with a common interest in advancing geoscientist’s ability to make
+effective use of modern, open tools and technologies to support
+their Data Science needs. ESDS welcomes participation from all
+NCAR/UCAR LCPOs. We are primarily a volunteer organization, and
```
Comment (dcherian) [inline about.md:7]:
```suggestion
NCAR/UCAR Lab/Centers/Programs/Offices (LCPOS). We are primarily a volunteer organization, and
```

**Review by dcherian — APPROVED** [review]

**Review by katiedagon — APPROVED** [review]

**Pull request merged**